### ISSUE TITLE
FEATURE: Support Ubuntu 20.04

### ISSUE THREAD

**Issue by dougburks** [Security-Onion-Solutions/securityonion#601]
https://ubuntu.com/blog/ubuntu-20-04-lts-arrives

**Comment by dougburks** [comment]
We'll have to wait until salt supports python 3.8:
https://github.com/saltstack/salt/issues/55310

**Comment by TOoSmOotH** [comment]
I have identified a salt issue that will most likely push this to post GA.

**Comment by ad0nis** [comment]
Has this been looked at again recently? I see that the referenced salt issue has been closed. -- Would love to get SO running on 20.04 LTS.

**Comment by GlennHD** [comment]
I'm interested in this as well. I've gotten most of my services migrated over to 20.04 LTS. SO2 is the only reason I still maintain an active 18.04 image =)

**Comment by jpvlsmv** [comment]
Looks like salt rev 3002.5 (currently being distributed by their repo.saltproject.io) is usable on 20.04, can the block be removed now?

**Comment by ad0nis** [comment]
I took a rough stab at this a few weeks ago. I was able to get to the point where the installer script would finish, but there are a few errors still. My changes didn't merge cleanly with updates made between then and now, but I took another stab at making the same changes last night, and submitted the following pull request as a move in the general direction of gaining ubuntu 20.04 support. 

https://github.com/Security-Onion-Solutions/securityonion/pull/3434
